packages/aws/src/application/AWSCredentialsProvider.ts

```diff
@@ -30,6 +31,11 @@ export default class AWSCredentialsProvider {
               configLoader().AWS.authentication.options.targetRoleName,
           },
         })
+      case 'IAM':
```

Comment:
Maybe `EC2-METADATA` is a better name for this credentials option? All the options have something to do with `IAM` right?